src/Custom.Build.props

```
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <MinVerMinimumMajorMinor>0.0</MinVerMinimumMajorMinor>
+    <MinVerMinimumMajorMinor>1.0</MinVerMinimumMajorMinor>
```

ClaimCheck 1.0 is the current version so this needs to be bumped past that:

```suggestion
    <MinVerMinimumMajorMinor>2.0</MinVerMinimumMajorMinor>
```

---

Resolved in [9793b15](https://github.com/Particular/NServiceBus.ClaimCheck/pull/76/commits/9793b15cd0eccb3e645bf4870389840fd1f291cd)